src/components/AddMoney/components/__tests__/MantecaPixQrDeposit.test.tsx

```
@@ -0,0 +1,77 @@
+jest.mock('@/components/Global/CopyToClipboard', () => ({
+    __esModule: true,
+    default: ({ textToCopy }: { textToCopy: string }) => <div data-testid="copy" data-text={textToCopy} />,
+}))
+jest.mock('@/components/Global/Icons/Icon', () => ({ Icon: () => <div /> }))
+jest.mock('@/components/0_Bruddle/Button', () => ({
+    Button: ({ children, onClick }: { children: React.ReactNode; onClick?: () => void }) => (
+        <button onClick={onClick}>{children}</button>
+    ),
+}))
+
+// eslint-disable-next-line import/first -- must come after jest.mock
```

ESLint

[error] Definition for rule 'import/first' was not found.

(import/first)